filter.txt

```diff
@@ -1251,3 +1251,5 @@ zoomtech.ir###text-3
 ||zoomit.ir/*.gif
 ||zoomit.ir/*/declarationPics/*.jpg
 ||zoomtech.ir/wp-content/uploads/*.gif
+||hiva.site/repo/tbb.mp4
+||*.namasha.com/videos/*.mp4
```

Comment:
you can remove the `*.` from the beginning. it would automatically match as a wildcard.